types/filenamify/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // This package uses the Function type, and it will take effort to fix.
```

Comment:
I just copied it from readme. Should I pull remove the file or delete only the line?